gammapy/stats/variability.py

```diff
@@ -349,16 +359,9 @@ def TimmerKonig_lightcurve_simulator(
 
     random_state = get_random_state(random_state)
 
-    if leakage_protection < 1.0:
-        raise ValueError("The leakage protection factor must be at least 1.0.")
+    npoints_ext = npoints * nchunks
```

Comment:
shouldn't there be a smiliar check for `nchunks`. I suppose 0.5 is a bad value, right?

Reply:
You should convert to int also.